resources/views/agreement/alreadysigned.blade.php

```diff
@@ -0,0 +1,18 @@
+@extends('layouts/app')
+
+@section('title')
+Membership Agreement | {{ config('app.name') }}
+@endsection
+
+@section('content')
+@component('layouts/title')
+  Membership Agreement
+@endcomponent
+
+<p>
+We already have a signed agreement on file for you for the most recent revision of the membership agreement.
```

Comment:
Might want to add a sentence explaining what the use should do next, even if it is "no action required"

I suspect the user might be somewhat confused if they ever end up here.